orcestra_book/intro.md

```diff
@@ -2,16 +2,36 @@
 
 **ORCESTRA - Organized Convection and EarthCare Studies over the Tropical Atlantic**
 
-The overarching objective of ORCESTRA is to better understand the physical mechanisms that organize tropical convection at the mesoscale, influencing the structure and dynamics of the inter-tropical convergence zone (ITCZ). This includes the interaction of convective organization with tropical waves and air-sea interaction, and the impact of convective organization on climate and the Earth’s radiation budget and processes of tropical cyclogenesis. In addition to advancing understanding of tropical meteorology and atmospheric processes, ORCESTRA observations will help calibrate and validate satellite remote sensing (especially EarthCARE) and a new generation of global ocean-eddy and storm-resolving climate models.
 
-ORCESTRA is an international initiative that combines different sub-campaigns:
+The overall objectives of the ORCESTRA campaign are summarised as followed:
+
+```{note}
+1. Determine the drivers of mesoscale organisation in the tropics and their impact on smallscale weather systems and the large scale circulation with a particular focus on the structure and variability of the Atlantic ITCZ
+2. Serve as a benchmark for satellite remote sensing and a new generation of high resolution storm resolving models
+```
+
+ORCESTRA is an international initiative that combines eight different sub-campaigns:
 * [BOW-TIE - Beobachtung von Ozean und Wolken – Das Trans ITCZ Experiment](bowtie.md)
 * [CELLO - Cloud and EarthCARE caL/vaL Observations](cello.md)
-* [CLARINET - CLoud and Aerosol Remote sensing for EarThcare](clarinet.md)
+* [CLARINET - Cloud and Aerosol Remote sensing for EarthCARE](clarinet.md)
 * [MAESTRO - Mesoscale organisation of tropical convection](maestro.md)
-* [PERCUSION - Persistent EarthCare underflight studies of the ITCZ and organized convection](percusion.md)
+* [PERCUSION - Persistent EarthCARE underflight studies of the ITCZ and organized convection](percusion.md)
 * [PICCOLO - Process Investigation of Clouds and Convective Organization over the atLantic Ocean](piccolo.md)
 * [SCORE - Sub-Cloud Observations of Rain Evaporation](score.md)
-* [STRINQS - Soundings and TuRbulent eddy measurements in the ITCZ with a Network of QuadcopterS](strinqs.md)
+* [STRINQS - Soundings and turbulent eddy measurements in the ITCZ with a network of quadcopters](strinqs.md)
```

Comment:
Changing the typing to a more readable words seems to be a good idea (although I think probably people had something in mind when coming up with the previous typing, but I can't really match it up with the abbreviated sub campaign names).